corehq/apps/app_manager/fields.py

```diff
@@ -377,12 +372,12 @@ def _get_applications_by_type(self, as_dict=True):
             (self.APP_TYPE_ALL, self._available_app_forms),
             (self.APP_TYPE_UNKNOWN, self._unknown_forms)
         )
-        _app_fmt = lambda c: (c[0], map(lambda f: RMIDataChoice(
+        _app_fmt = lambda c: (c[0], [RMIDataChoice(
             f['app']['id'] if f.get('has_app', False) else self.UNKNOWN_SOURCE,
             f['app']['name'] if f.get('has_app', False) else _("Unknown Application"),
             f
-        ), c[1]))
-        apps_by_type = map(_app_fmt, apps_by_type)
+        ) for f in c[1]])
+        apps_by_type = list(map(_app_fmt, apps_by_type))
```

Comment:
ha, I didn't look at the context carefully enough. I thought `apps_by_type` was being referenced in multiple places (it is), but didn't realize it was being reassigned. `dict(map(...))` should be fine.